File: Class_Practice/class_teat_practice/code1.c
#include <stdio.h>
#include <math.h>

int main()
{
    int a, b;
    scanf("%d %d", &a, &b);
    int sqr = pow((a + b), 2);
    printf("Squire of  (a+b): %d", sqr);
    return 0;
}

File: Class_Practice/class_teat_practice/Code1.go
package main

import (
	"fmt"
	// "math"
)

func main() {
	var a, b, sum int
	fmt.Scanf("%d %d", &a, &b)
	sum = a + b
	// var multi int = 1
	multi := 1
	for i := 0; i < 3; i++ {
		multi = sum*multi
	}
	fmt.Printf("Sqr of (a+b); %d", multi)
}
